CPAC/registration/tests/test_fsl_registration_connector.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from CPAC.registration.registration import FSL_registration_connector
+
+
+class AttrDict(dict):
+    def __getattr__(self, item):
+        value = self[item]
+        if isinstance(value, dict):
+            return AttrDict(value)
+        return value
```

Comment:
Either 

```suggestion
from CPAC.utils.configuration import Configuration
```

or just define `AttrDict` in one place and import it where we use it